test/testcafe/spec/EnrollAuthenticatorPhone_spec.js

```diff
@@ -0,0 +1,120 @@
+  const enrollPhonePage = await setup(t);
+
+  // Switch to Voice
+  await enrollPhonePage.clickRadio();
+  await t.expect(enrollPhonePage.getSaveButtonLabel()).eql('Send a code via voice call');
+
+  // Extension field is shown
+  await t.expect(enrollPhonePage.extensionIsHidden()).eql(false);
+  const extensionText = await enrollPhonePage.getElement('.phone-authenticator-enroll__phone-ext').innerText;
+  await t.expect(extensionText.trim()).eql('Extension');
+
+  // Default country code US
+  const countryCodeText = await enrollPhonePage.getElement('.phone-authenticator-enroll__phone-code').innerText;
+  await t.expect(countryCodeText.trim()).eql('+1');
+
+  // Phone Number input field is rendered small
+  await t.expect(enrollPhonePage.phoneNumberFieldIsSmall()).eql(true);
+});
+
+test.requestHooks(mock)(`phone number is required`, async t => {
+  const enrollPhonePage = await setup(t);
+  // fields are required
+  await t.expect(enrollPhonePage.hasPhoneNumberError()).eql(false);
+  await enrollPhonePage.clickSaveButton();
+  await enrollPhonePage.waitForError();
+  await t.expect(enrollPhonePage.hasPhoneNumberError()).eql(true);
+});
+
+test.requestHooks(mock)(`should succeed when values are filled in sms mode`, async t => {
+  const enrollPhonePage = await setup(t);
+  const successPage = new SuccessPageObject(t);
+
+  await enrollPhonePage.fillPhoneNumber('4156669999');
+  await enrollPhonePage.clickSaveButton();
+
+  const pageUrl = await successPage.getPageUrl();
+  await t.expect(pageUrl)
+    .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
+});
+
+test.requestHooks(mock)(`should succeed when values are filled in voice mode`, async t => {
+  const enrollPhonePage = await setup(t);
+  const successPage = new SuccessPageObject(t);
+  await enrollPhonePage.clickRadio();
+  await enrollPhonePage.fillPhoneNumber('4156669999');
+  await enrollPhonePage.clickSaveButton();
+
+  const pageUrl = await successPage.getPageUrl();
+  await t.expect(pageUrl)
+    .eql('http://localhost:3000/app/UserHome?stateToken=mockedStateToken123');
+});
+
+
+/* Voice only option mocks */
+test.requestHooks(voiceOnlyOptionMock)(`default is voice mode`, async t => {
+  const enrollPhonePage = await setup(t);
+
+  // Check title
+  await t.expect(enrollPhonePage.getFormTitle()).eql('Set up phone authentication');
+  await t.expect(enrollPhonePage.getFormSubtitle())
+    .eql('Set up verification with a phone number. You will receive a code sent to your phone');
+  await t.expect(enrollPhonePage.getSaveButtonLabel()).eql('Send a code via voice call');
+
+  // Extension field is not hidden
+  await t.expect(enrollPhonePage.extensionIsHidden()).eql(false);
+  // Phone field is rendered small
+  await t.expect(enrollPhonePage.phoneNumberFieldIsSmall()).eql(true);
+});
+
+test.requestHooks(voiceOnlyOptionMock)(`should succeed when values are filled when in voice only mode`, async t => {
```

Comment:
maybe merge into the testcase above (line 96)?

Reply:
I was thinking to keep it granular for two reasons -
1. The test case on #96 would be too big and multiple things will happen in it.
2. The granularity also helps have it's own test case for any future requirements that we may have for voice-only mode.